app/views/provider_interface/application_choices/new_reject.html.erb

```diff
@@ -0,0 +1,88 @@
+<%= form_tag provider_interface_application_choice_new_confirm_reject_path, method: :get do %>
+  
+  <fieldset class="govuk-fieldset">
+  
+  <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl govuk-!-margin-bottom-8">
+  
+    <h1 class="govuk-fieldset__heading">
+      
+    <span class="govuk-caption-xl">Application for Upper Primary with Modern Foreign Languages (XR9C)</span>
+    Reject application
+  
+    </h1>
+  
+  </legend>
+ 
+<dl class="govuk-summary-list govuk-!-margin-bottom-8">
```

Comment:
We can use the `SummaryCardComponent` for this probably:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/37da4915ad1ae9762e5e7b449066b1ecb9cbf41c/app/components/summary_card_component.rb

Reply:
Works great. Changed it to support a borderless version